tests/nat/tools/test_code_execution_sandbox.py

```
@@ -0,0 +1,447 @@
+    assert "Iteration 1" in result["stdout"]
+    assert "Iteration 2" in result["stdout"]
+    assert "Completed" in result["stdout"]
+    assert result["stderr"] == ""
+
+
+@pytest.mark.integration
+def test_missing_generated_code_field(sandbox_config):
+    """Test request missing the generated_code field."""
+    payload = {"timeout": 10, "language": "python"}
+
+    response = requests.post(sandbox_config["execute_url"], json=payload)
```

Ruff (0.14.3)

Probable use of `requests` call without timeout

(S113)